src/NuGet/Microsoft.Orleans.Development.nuspec

```diff
@@ -1,34 +0,0 @@
-﻿<?xml version="1.0"?>
```

Comment:
Do you think it's better to duplicate the content of this package in two, and potentially more in the future, other packages?